src/RestApi.php

```diff
@@ -84,24 +84,33 @@ public static function maybe_init_cart_session( $return, $request = false ) {
 	 * @return array
 	 */
 	protected static function get_controllers() {
-		return [
+		$controllers = [
 			'product-attributes'            => __NAMESPACE__ . '\RestApi\Controllers\ProductAttributes',
 			'product-attribute-terms'       => __NAMESPACE__ . '\RestApi\Controllers\ProductAttributeTerms',
 			'product-categories'            => __NAMESPACE__ . '\RestApi\Controllers\ProductCategories',
 			'product-tags'                  => __NAMESPACE__ . '\RestApi\Controllers\ProductTags',
 			'products'                      => __NAMESPACE__ . '\RestApi\Controllers\Products',
 			'variations'                    => __NAMESPACE__ . '\RestApi\Controllers\Variations',
 			'product-reviews'               => __NAMESPACE__ . '\RestApi\Controllers\ProductReviews',
-			'store-cart'                    => __NAMESPACE__ . '\RestApi\StoreApi\Controllers\Cart',
 			'store-cart-items'              => __NAMESPACE__ . '\RestApi\StoreApi\Controllers\CartItems',
 			'store-cart-coupons'            => __NAMESPACE__ . '\RestApi\StoreApi\Controllers\CartCoupons',
 			'store-cart-shipping-rates'     => __NAMESPACE__ . '\RestApi\StoreApi\Controllers\CartShippingRates',
-			'store-cart-order'              => __NAMESPACE__ . '\RestApi\StoreApi\Controllers\CartOrder',
 			'store-customer'                => __NAMESPACE__ . '\RestApi\StoreApi\Controllers\Customer',
 			'store-products'                => __NAMESPACE__ . '\RestApi\StoreApi\Controllers\Products',
 			'store-product-collection-data' => __NAMESPACE__ . '\RestApi\StoreApi\Controllers\ProductCollectionData',
 			'store-product-attributes'      => __NAMESPACE__ . '\RestApi\StoreApi\Controllers\ProductAttributes',
 			'store-product-attribute-terms' => __NAMESPACE__ . '\RestApi\StoreApi\Controllers\ProductAttributeTerms',
 		];
+
+		$experimental_controllers = [
+			'store-cart'       => __NAMESPACE__ . '\RestApi\StoreApi\Controllers\Cart',
+			'store-cart-order' => __NAMESPACE__ . '\RestApi\StoreApi\Controllers\CartOrder',
+		];
```

Comment:
@mikejolley those controllers only affect the cart and checkout endpoints right?

Reply:
They are built for that purpose yes. Should ensure these files are included during unit tests though.

Reply:
I don't think it's a big deal if the REST endpoints are gated or not as they are explicitly listed as being private anyways.  So I would recommend just not gating the endpoints. If you decide to gate them, then you'll need to gate `store-cart-items`, `store-cart-coupons`, `store-cart-shipping-rates` and `store-customer` too.